### ISSUE TITLE
Certain text search queries cause an SQLite error

### ISSUE THREAD

**Issue by fechan** [eba-diary/Travelogues-webapp#11]
Searching `Nile to Aleppo with the Light-horse in the Middle-East.` in title (and maybe other text fields as well) causes the following error:
```
[Error: SQLITE_ERROR: no such column: horse] {
  errno: 1,
  code: 'SQLITE_ERROR'
}
```

Running the following SQL query in isolation throws the same error:
```sql
SELECT rowid FROM publicationsfts WHERE title MATCH "Nile to Aleppo with the Light-horse in the Middle-East."
```

**Comment by fechan** [comment]
It's an FTS syntax escaping problem. [https://stackoverflow.com/questions/28971633/how-to-escape-string-for-sqlite-fts-query](https://stackoverflow.com/questions/28971633/how-to-escape-string-for-sqlite-fts-query)

**Comment by fechan** [comment]
Fixed in 7dac9d5162f652b62c276574b9c77933369ad777
